src/Symfony/Component/Notifier/Bridge/Smsbox/Tests/SmsboxTransportTest.php

```diff
@@ -206,17 +206,18 @@ public function testSmsboxOptionsInvalidDateTimeAndDate()
             return $response;
         });
 
+        $dateTime = new \DateTimeImmutable('+1 day');
+
         $this->expectException(\InvalidArgumentException::class);
         $this->expectExceptionMessage("Either Symfony\Component\Notifier\Bridge\Smsbox\SmsboxOptions::dateTime() or Symfony\Component\Notifier\Bridge\Smsbox\SmsboxOptions::date() and Symfony\Component\Notifier\Bridge\Smsbox\SmsboxOptions::hour() must be called, but not both.");
-        $dateTime = \DateTimeImmutable::createFromFormat('d/m/Y H:i', '01/11/2024 18:00', new \DateTimeZone('UTC'));
         $message = new SmsMessage('+33612345678', 'Hello');
 
         $smsboxOptions = (new SmsboxOptions())
             ->mode(Mode::Expert)
             ->sender('SENDER')
             ->strategy(Strategy::Marketing)
             ->dateTime($dateTime)
-            ->date('01/01/2024');
```

Comment:
Previously this date was different than the one passed to `dateTime()`. Now both are the same. Is that an issue?

Reply:
No, we just assert that calling `dateTime()` and `date()` throws, but the actual data doesn't matter.